controllers/main.js

```diff
@@ -8,7 +8,7 @@ module.exports = {
          socketToken: socketAuthenticator.getTokenForUser(req.user),
          socketVersion: socketio,
          user: req.user.username,
-         debug: config.debug
+         debugTools: true,
```

Comment:
What about
```suggestion
         debugTools: config.debug,
```
so it can still be configured?